_posts/2025-01-29-cockpit-332.md

```diff
@@ -0,0 +1,79 @@
+
+## Packageless session for all supported operating systems
+
+[Cockpit 327](https://cockpit-project.org/blog/cockpit-327.html) introduced support for connecting to remote machines without any installed Cockpit packages. That only worked if the local and remote machine had the same operating system version. This version extends this "packageless session" feature to all operating systems which Cockpit [continuously tests](https://cockpit-project.org/running.html#installation--setup). At the time of writing this includes:
+
+  * Arch Linux
+  * CentOS Stream 9, 10
+  * Debian 12, testing/unstable
+  * Fedora Linux 40, 41, 42 (rawhide)
+  * Red Hat Enterprise Linux 8, 9, 10
+  * Ubuntu 22.04 LTS, 24.04 LTS, 24.10
+
+This list will change over time, along with new operating system releases and their end-of-lives.
+
+## Show system boot in metrics
+
+![screenshot of ## show system boot in metrics]({{ site.baseurl }}/images/332-show-system-boot-in-metrics.png)
+
+## Machines: "Launch viewer" now uses correct address
+
+The "Launch viewer" button in the "Console" card of a virtual machine would tell the remote viewer application to connect to the listening address of the VNC server. This is wrong when the server listens on 127.0.0.1 (libvirt default) or 0.0.0.0 (allow connections on all interfaces). Now Cockpit will tell the remote viewer application to connect to the same domain name or IP address that Cockpit itself is using to reach the virtual machine host.
+
+## Podman: Automatically start podman.socket
+
+If the system or user's `podman.socket` unit isn't running, cockpit-podman now silently starts it on its own, instead of asking the user about it. There is no need to have the service enabled (i.e. always start it on boot or user login) for cockpit-podman any more.
+
+## Podman: Improve quadlet compatibility
+
+Podman quadlet's are now visually identified by a `service` label in the containers overview and the rename a container option is now also hidden for quadlets.
+
+![screenshot of improve quadlet compatibility]({{ site.baseurl }}/images/332-podman-improve-quadlet-compatibility.png)
+
+## Files: Show user:group in the footer
+
+The footer now shows the user and group of the current directory.
+
+![screenshot of group in the footer]({{ site.baseurl }}/images/332-files-show-user-group-in-the-footer.png)
+
+## Try it out
+
+Cockpit 332, cockpit-machines 327, cockpit-podman 100, and cockpit-files 15 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/332)
+* [Cockpit Fedora 41](https://bodhi.fedoraproject.org/updates/?releases=F41&packages=cockpit)
```

Comment:
standard reminder thread: update URLs as packages become available on bodhi.

Reply:
Yup, just done but bodhi update for cockpit-machines F40 is missing